package.json

```diff
@@ -53,7 +53,14 @@
       "fast-loops@<1.1.4": ">=1.1.4",
       "fast-xml-parser@<4.4.1": ">=4.4.1",
       "elliptic@<6.5.7": ">=6.5.7",
-      "micromatch@<4.0.8": ">=4.0.8"
+      "micromatch@<4.0.8": ">=4.0.8",
+      "webpack@>=5.0.0-alpha.0 <5.94.0": ">=5.94.0",
+      "send@<0.19.0": ">=0.19.0",
+      "serve-static@<1.16.0": ">=1.16.0",
+      "path-to-regexp@>=2.0.0 <3.3.0": ">=3.3.0",
+      "vite@>=5.3.0 <5.3.6": ">=5.3.6",
+      "vite@>=5.3.0 <=5.3.5": ">=5.3.6",
```

Comment:
This seems a bit unecessary to have both? Don't they do the same thing?

Reply:
Also, since `vite` is in our devDependencies, why not update it there directly and instead override it?

Reply:
Done!